org.musicbrainz.Picard.json

```diff
@@ -153,7 +221,7 @@
             "name": "python-pkgconfig",
             "buildsystem": "simple",
             "build-commands": [
-                "python2 setup.py install --prefix=/app"
+                "python3 setup.py install --prefix=/app"
```

Comment:
The Python packaging ecosystem is slowly but steadily moving towards using Pip for everything, with Pip becoming a sort of "meta" tool knowing how to handle distutils, setuptools, flit, …

As a result, I prefer using the following for Python modules: `pip3 install --no-deps --prefix=/app .`.

Unfortunately that doesn't work with Python2 (the `pip` bundled for Python2 in the various SDKs is too old), and it also doesn't work for Picard 2.0 with Python 3 (their setup.py file is so convoluted, it breaks some assumptions from Pip… this might eventually get fixed in the future)

I'd much prefer we used Pip everywhere possible, and keep the setuptools-specific instructions only when absolutely necessary (the "picard" module itself)